script.js

```diff
@@ -127,9 +127,22 @@ function renderQuestion() {
 }
 
 function endGame() {
-  console.log('game ended');
+  document.querySelector('.quiz-container').remove();
+  var d = document.getElementById("game-container")
+  d.className='end-game';
+  var e = document.getElementById("game-score")
+  e.style.color='black';
```

Comment:
everything which is style based needs to be done in css file. Just add the classes via JS if necessary.